### ISSUE TITLE
js-yaml throws "unknown tag" warning when using peek()

### ISSUE THREAD

**Issue by BrennanRoberts** [Oblong/plasma-js-bridge#3]
When using peek, js-yaml throws the warning
```
JS-YAML: unknown tag !<tag:oblong.com,2009:slaw/protein> at line ..., column ...
```
when a protein is received.

We can:

1. Create a schema for the slaw type and pass it when calling `yaml.safeLoadAll(...)` 
1. Silence all warnings from js-yaml by giving it our own `onWarning(noOp)`

**Comment by tomjakubowski** [comment]
A client had started down this path with their own kinda-equivalent to this library.  Let me see what I can dig up.

**Comment by BrennanRoberts** [comment]
I've got a simple patch that does 1

**Comment by tomjakubowski** [comment]
closed by #4
